### PR TITLE
Create domain mohammedkhc and aliases for it.

### PR DESCRIPTION
**Pull request by MohammedKHC**
## Website Link/Preview
https://mohammedkhc.pages.dev

### CODE REVIEW

File: domains/khc.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MohammedKHC0",
+        "email": "mohammedkhcool@gmail.com"
+    },
+    "record": {
+        "URL": "mohammedkhc.is-a.dev"
```
Comment (CuteDog5695) [inline domains/khc.json:7]: URL record needs to have protocol. (aka https://)

File: domains/mhmd.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MohammedKHC0",
+        "email": "mohammedkhcool@gmail.com"
+    },
+    "record": {
+        "URL": "mohammedkhc.is-a.dev"
```
Comment (CuteDog5695) [inline domains/mhmd.json:7]: URL record needs to have protocol. (aka https://)

File: domains/mohammed.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MohammedKHC0",
+        "email": "mohammedkhcool@gmail.com"
+    },
+    "record": {
+        "URL": "mohammedkhc.is-a.dev"
```
Comment (CuteDog5695) [inline domains/mohammed.json:7]: URL record needs to have protocol. (aka https://)

File: domains/muhammad.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MohammedKHC0",
+        "email": "mohammedkhcool@gmail.com"
+    },
+    "record": {
+        "URL": "mohammedkhc.is-a.dev"
```
Comment (CuteDog5695) [inline domains/muhammad.json:7]: URL record needs to have protocol. (aka https://)

File: mohamad.json
Comment (CuteDog5695) [inline mohamad.json:1]: File isn't in json format.

File: mohamad.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MohammedKHC0",
+        "email": "mohammedkhcool@gmail.com"
+    },
+    "record": {
+        "URL": "mohammedkhc.is-a.dev"
```
Comment (CuteDog5695) [inline mohamad.json:7]: URL record needs to have protocol. (aka https://)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by MohammedKHC0** [comment]
i resolved the issues, please recheck

File: mohamad.json
Comment (MohammedKHC0) [reply mohamad.json:1]: this file is already registered by someone else so please ignore it, I dont know how.

Comment (CuteDog5695) [reply mohamad.json:1]: > this file is already registered by someone else so please ignore it, I dont know how.

Then you can't claim the domain, plain and simple.

**Comment by CuteDog5695** [comment]
For mohamad.json since its taken you'd need to delete it.

**Review by iostpa — APPROVED** [review]
Seems good to me ig (also if the domains don't redirect I recommend you use [redirect.pizza](https://redirect.pizza))

**Comment by MohammedKHC0** [comment]
Thanks for your time, And for the fast answer, I really appreciate it.

**Pull request merged**